bundles/org.openhab.binding.yamahamusiccast/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -98,7 +98,7 @@
 	<channel-type id="volumeDB">
 		<item-type>Number:Dimensionless</item-type>
 		<label>Volume in dB</label>
-		<description>Volume channel - in decibel (dB) (-90 dB if not available)</description>
+		<description>Volume channel - in decibel (dB)</description>
```

Comment:
Perhaps "Volume level" instead of "Volume channel"? "Channel" seems to be redundant as channel description. The same goes for other channels.